src/InMemoryEventStore.php

```diff
@@ -151,6 +152,10 @@ public function load(
             }
         }
 
+        if (count($streamEvents) == 0) {
+            throw NoNewerEventsFound::with($streamName);
```

Comment:
I have tried to write a test to increase the linecoverage here:
https://coveralls.io/builds/10114395/source?filename=src%2FProjection%2FInMemoryEventStoreReadModelProjection.php#L298
As I could see, there was no test possible to throw this exception.
So my idea war to add an eventsNotFound exception instead and catch it.